spec/requests/api/v1/topics/retrieve_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'GET api/v1/topics', type: :request do
+  let(:password) { 'password' }
+  let(:user) { create(:user, password: password) }
```

Comment:
Indent variables

Reply:
It's a good practice to name these files as the method we are trying to test. In this case, as you are trying to fetch topics, you should call it "index_spec.rb".